.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @dbl-works/dev-ops-admin
```

Comment:
we need to backport PR's from a branch that has this file removed, see: https://github.com/marcqualie/ecsx/pull/84

--> Maybe we open a branch on our fork specific for that, and name it e.g. `backport`?

Reply:
Dropped this on our fork and replaced functionality by manually adding people in the settings to ease upstreaming.